core/src/core_tests/workflow_tasks.rs

```diff
@@ -2761,13 +2761,13 @@ async fn use_compatible_version_flag(
             let can_cmd = c.commands.pop().unwrap().attributes.unwrap();
             match can_cmd {
                 Attributes::ContinueAsNewWorkflowExecutionCommandAttributes(a) => {
-                    assert_eq!(a.use_compatible_version, compat_flag_expected);
+                    assert_eq!(a.inherit_build_id, compat_flag_expected);
                 }
                 Attributes::ScheduleActivityTaskCommandAttributes(a) => {
-                    assert_eq!(a.use_compatible_version, compat_flag_expected);
+                    assert_eq!(a.use_workflow_build_id, compat_flag_expected);
```

Comment:
`use_compatible_version` has changed to `inherit_build_id` in some places and `use_workflow_build_id` in others?

Reply:
Yes, unfortunately. I didn't quite track that in the API changes. It does make some logical sense, but being consistent would've been nice. Oh well, whatever, we can make it consistent in the SDK apis.